changelog.d/6345.doc

```diff
@@ -0,0 +1 @@
+

```

Comment:
there's not much point in an empty changelog file. Indeed, it will mess up our changelog on the next release.

There's a valid argument for merging documentation changes like this straight to master without a release or changelog entry, but in that case, you need to omit the changelog.d file.